spec/02-integration/14-tracing/01-instrumentations_spec.lua

```diff
@@ -160,7 +160,7 @@ for _, strategy in helpers.each_strategy() do
         -- Making sure it's alright
         local spans = cjson.decode(res)
         assert.is_same(4, #spans, res)
-        assert.is_same("GET /", spans[1].name)
+        assert.is_same("kong.request", spans[1].name)
```

Comment:
let's rename to `kong` as discussed above
```suggestion
        assert.is_same("kong", spans[1].name)
```